_build/html/contents.html

```diff
@@ -6,7 +6,7 @@
   <head>
```

Comment:
Oh yea, I think I just pushed my entire working directory to this repository thinking that it's a temporary place to test and experiment before integrating with the other user docs or whatnot. I've put on pause any thinking around where this API information should live so that I could get on with the restructuring and editing of content. In my haste, I pushed everything, but totally valid points Ross. :) I will look into adding a gitignore file! Pretty sure we only need the *.rst, doctree, and any other environment files.